Remotion/Web/Core/UI/Controls/ListMenuImplementation/Rendering/ListMenuRenderer.cs

```diff
@@ -74,10 +74,31 @@ public void Render (ListMenuRenderingContext renderingContext)
       RegisterMenuItems(renderingContext);
 
       renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Id, renderingContext.Control.ClientID);
-      renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Class, CssClassListMenu);
-      renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute2.Role, HtmlRoleAttributeValue.Menu);
+      renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Class, "listMenuRoot");
```

Comment:
```suggestion
      renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Class, CssClassListMenu);
```
Root elements need to stick to the "official" classes. 

Please add a breaking change for the updated root level element.